src/engraving/libmscore/clef.cpp

```diff
@@ -311,7 +311,7 @@ Element* Clef::drop(EditData& data)
             if (segm->element(track())) {
                 score()->undoRemoveElement(segm->element(track()));
             }
-            Ambitus* r = new Ambitus(score());
+            Ambitus* r = new Ambitus(segm);
             r->setParent(segm);
```

Comment:
Is this still needed?

Reply:
Ah, yes, I indeed realized that after seeing the `isParentSetExplicitly` stuff. Thanks!